pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
 	core_v1 "k8s.io/api/core/v1"
```

Comment:
```GO
	// The last error encountered during create snapshot operation, if any.
	// This field must only be set by the entity completing the create snapshot		// This field must only be set by the entity completing the create snapshot
	// operation, i.e. the external-snapshotter.		// operation, i.e. the external-snapshotter.
	// +optional		// +optional
	Error *storage.VolumeError `json:"error,omitempty" protobuf:"bytes,4,opt,name=error,casttype=VolumeError"`		Error *storage.VolumeError `json:"error,omitempty" protobuf:"bytes,4,opt,name=error,casttype=VolumeError"`
```

Maybe rename to something that implies that this is a terminal or fetal error.

Reply:
no longer a concern if we do retry